plugins/woocommerce/tests/e2e-pw/tests/merchant/settings-tax.spec.js

```diff
@@ -29,7 +29,7 @@ test.describe( 'WooCommerce Tax Settings > enable', () => {
 	} );
 } );
 
-test.describe( 'WooCommerce Tax Settings', () => {
+test.describe.serial( 'WooCommerce Tax Settings', () => {
```

Comment:
The tests in this spec heavily depend on each other, so I had to use `serial` here.